src/components/course/data/hooks.jsx

```diff
@@ -724,8 +724,10 @@ export const useUserSubsidyApplicableToCourse = ({
       const result = await getApplicableSubsidyForCourse();
       if (result.applicableSubsidy) {
         setUserSubsidyApplicableToCourse(result.applicableSubsidy);
+        setMissingUserSubsidyReason(null);
```

Comment:
I wonder if we should just fully reset this data to a zero state elsewhere? This will only set it after a number of API calls are being made. Can/Should we set this at an earlier state in the processing of this button?

Reply:
You're right, so we added an `isLoadingAny` to wait for the loading to finish before we show anything